server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -245,15 +245,25 @@ class LivyConf(loadDefaults: Boolean) extends ClientConf[LivyConf](null) {
   /** Return the spark deploy mode Livy sessions should use. */
   def sparkDeployMode(): Option[String] = Option(get(LIVY_SPARK_DEPLOY_MODE)).filterNot(_.isEmpty)
 
+  /** Return the  spark home version */
+  def SPARK_HOME_VER(version: String): Option[String] = {
```

Comment:
If I'm understanding you correctly, that's where my comment below would fix it. I'll edit the comment below to clarify it.